### PR TITLE
RHINENG-14585: Remove pending alerts from processing and add another value to the unique key

### PR DESCRIPTION
**Pull request by Fewwy**
https://issues.redhat.com/browse/RHINENG-14585

### CODE REVIEW

File: web/src/components/Incidents/processAlerts.js [hunk 1]
```diff
@@ -27,10 +27,13 @@
  * // Returns an array where the two alerts are grouped together with deduplicated values.
  */
 export function groupAlerts(objects) {
+  // Step 1: Filter out objects with alertstate 'pending'
+  const filteredObjects = objects.filter((obj) => obj.metric.alertstate !== 'pending');
```
Comment (falox) [inline web/src/components/Incidents/processAlerts.js:31]: better `=== 'firing'`

**Review by falox — CHANGES_REQUESTED** [review]

File: web/src/components/Incidents/processAlerts.js (hunk 1 above)
Comment (Fewwy) [reply web/src/components/Incidents/processAlerts.js]: Done

**Pull request merged**